Add examples for whitelist_external_dirs

cooljimy84 opened the pull request:
Added examples for whitelist_external_dirs as I found it difficult to figure out 
a) it had to follow homeassistant:
b) drop the last slash for the directory i.e. /tmp/exampledir NOT /tmp/exampledir/

**Description:**


**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

cooljimy84 committed:
Add examples for whitelist_external_dirs

Added examples for whitelist_external_dirs as I found it difficult to figure out 
a) it had to follow homeassistant:
b) drop the last slash for the directory i.e. /tmp/exampledir NOT /tmp/exampledir/

mf-social commented:
Why is one of the examples an internal directory?

I was of the understanding that you only have to whitelist **external** directories, hence the name?

cooljimy84 commented:
Your right, it's only outside of the running folder, I'll change the example

cooljimy84 committed:
updated the example

I changed one of the example whitelists, as it was an already white listed directory.

cooljimy84 reviewed (commented):
Changed the example

DubhAd reviewed (approved):
👍
